bitnami/kibana/values.yaml

```
@@ -162,6 +162,9 @@ savedObjects:
   ## @param savedObjects.configmap Configmap containing NDJSON files to be imported during Kibana initialization (evaluated as a template)
   ##
   configmap: ""
+  ## @param savedObjects.SavedObjectsOverWrite Overwrite saved objects
+  SavedObjectsOverWrite: false
```

I think this name is clearer:

```suggestion
  overwrite: false
```